### DISCUSSION TITLE
High CPU Usage

### DISCUSSION THREAD

**Discussion by miroljub1995** [community/community#14026] [Codespaces] [answered]
When I open Pull Request in github.dev sometimes CPU Usage is high. It looks like some infinite loop.

Here is a result from Profiler, maybe that helps:
![image](https://user-images.githubusercontent.com/46632388/161227325-421383fd-7ac4-4eb3-92e6-fc538e4487b4.png)

**Comment by slpkbt** [comment]
same here, sometimes for some reasons copilot overloads cpu

**Reply by AnsonH** [reply]
Same for me. I'm getting 65% of CPU utilization when I open github.dev to view a file (not even editing).

**Comment by Chiroyce1** [comment]
Can repro, Firefox on macOS. It used to be very little before but now it goes up to 20-30% when I'm even viewing a file!

**Comment by TapGhoul** [comment]
Have been seeing this too. It seems to be coming from repeatedly trying to reauth with github in a tight loop from my end - shows in the console too.

I see
```
vscs:web-client:0446 Found password in local storage vsonline.keytar.vscodevscode.github-authentication.github.auth
vscs:web-client:0446 Found password in partner info strategy as well. Merging results.
vscs:web-client:0446 Saving password in local storage Object { service: "vscodevscode.github-authentication", account: "github.auth" }
vscs:web-client:0446 Responding to VSCode keytar-shim request. Object { service: "vscodevscode.github-authentication", account: "github.auth" }
```

in a tight loop.

**Reply by TapGhoul** [reply]
This issue has rendered spaces unusable for the past week or 2, by the way.

**Comment by Chiroyce1** [comment, marked as answer]
I think it's fixed now..
